cekit/cli.py

```diff
@@ -125,6 +125,21 @@ def build_buildah(ctx, pull, tags):  # pylint: disable=unused-argument
     run_build(ctx, 'buildah')
 
 
+@build.command(name="podman", short_help="Build using Podman engine")
+@click.option('--pull', help="Always try to fetch latest base image.", is_flag=True)
+@click.option('--tag', 'tags', metavar="TAG", help="Use specified tag to tag the image after build, can be specified multiple times.", multiple=True)
+@click.pass_context
+def build_podman(ctx, pull, tags):
```

Comment:
You can also add here this:

```suggestion
def build_podman(ctx, pull, tags):   # pylint: disable=unused-argument
```

Reply:
Thanks!